prow/cmd/sinker/main.go

```diff
@@ -59,20 +57,21 @@ func main() {
 	pkc := kc.Namespace(configAgent.Config().PodNamespace)
 
 	// Clean now and regularly from now on.
-	clean(kc, pkc)
-	t := time.Tick(period)
+	clean(kc, pkc, configAgent)
+	t := time.Tick(configAgent.Config().Sinker.ResyncPeriod)
```

Comment:
This won't be reread if we update the `ResyncPeriod`.  I suggest rereading the period every loop and manually waiting the appropriate amount of time.